_data/tools.json

```diff
@@ -1816,5 +1816,14 @@
       "platforms": "All Web Applications Supported",
       "note": "Automatic Penetration Testing for Web Applications & API Schema Penetration Testing",
       "type": "DAST"
-   }  
+   },
+   {
+      "title": "Lucent Sky AVM",
+      "url": "https://www.lucentsky.com/avm",
+      "owner": "Lucent Sky",
+      "license": "Commercial",
+      "platforms": "SaaS or On-Premises",
+      "note": "Automatic finds and fixes application vulnerabilities in source code. Supported .NET, ASP, Android, C and C++, ECMAScript, Go, iOS, Java, PHP, Pytohn, Ruby, and Visual Basic applications.",
```

Comment:
Automatically?

Reply:
The tool finds vulnerabilities in source code and generates source code patches to replace vulnerable source code to fix the vulnerabilities.

If automatically is not an ideal word to use in this context, does the above sound better?

Reply:
I was suggesting you change “Automatic” ➡️ “Automatically”

Reply:
Must have made the error while copy-n-pasting it 😳. Thanks for catching it!